core/classes/Database/DB.php

```diff
@@ -92,15 +117,13 @@ public function transaction(Closure $closure) {
         $result = null;
```

Comment:
This `transaction` method should probably just be dropped, not being used anywhere, and now that you've exposed those PDO things, they are much better to use

Reply:
On the flipside is it worth keeping for future use since it saves you having to begin, try/catch with a commit/rollback each time?